src/pgrn-query-expand.c

```diff
@@ -502,7 +503,19 @@ pgroonga_query_expand(PG_FUNCTION_ARGS)
 	}
 	tableOID = DatumGetObjectId(tableOIDDatum);
 	currentData.table = RelationIdGetRelation(tableOID);
-
+#ifdef PGRN_HAVE_RELKIND_HAS_TABLE_AM
+	if(!RELKIND_HAS_TABLE_AM(currentData.table->rd_rel->relkind))
+#else
+	if((currentData.table->rd_rel->relkind != RELKIND_RELATION)
+	   && (currentData.table->rd_rel->relkind != RELKIND_TOASTVALUE)
+	   && (currentData.table->rd_rel->relkind != RELKIND_MATVIEW))
```

Comment:
We should define `PGRN_RELKIND_HAS_TABLE_AM()` macro instead.
We already use this approach for other macros. See `pgrn-compatible.h`.